clancy_database/management/commands/load_icon_data.py

```diff
@@ -0,0 +1,46 @@
+import csv
+from django.core.management.base import BaseCommand, CommandError
+from django.conf import settings
+import os.path
+import clancy_database
+from clancy_database.spreadsheet import insert_db_columns
+import sqlite3
+import time
+
+
+class Command(BaseCommand):
+    help = 'Inserts New Columns into the database via additional_columns.sql file'
```

Comment:
Note that since the columns are already being added to the `lemma` table in `schema.sql`, it may not be necessary to use `additional_columns.sql` if the database is re-created from scratch (see [README](https://github.com/Harvard-ATG/visualizing_russian_tools#source-data) for instructions). Then it would be more a matter of loading the data from `icons.csv` after the core database tables have been created by the  `create_clancy_db` management command. This loading step could then be part of the overall process of refreshing the database (e.g. create DB from spreadsheet, load data, dump snapshot of database).

If you were to go that path, you'd have to add an additional step to the README:

```
$ ./manage.py convert_clancy_xls --xlsfile NewVisualizingRussian.xls --csvfile russian.csv
$ ./manage.py create_clancy_db --csvfile russian.csv --dbfile russian.sqlite3
$ ./manage.py load_sharoff_data --dbfile russian.sqlite3
$ ./manage.py load_rnc_data --dbfile russian.sqlite3
$ ./manage.py load_icon_data --dbfile russian.sqlite3  <--- add this step
$ sqlite3 russian.sqlite3 .dump > russian.sql
$ gzip russian.sql && mv russian.sql.gz clancy_database/data/russian.sql.gz
```

The end result is a gzipped snapshot of the database (`russian.sql.qz`) which is ultimately what we want, since that's what is used by the `import_clancy_sqldump` management command to load everything after the django migrations have been run.

As discussed, there may be an alternative for the loading step with django migrations, but that may be a larger story.